### PR TITLE
[VI] Update `Twitter`

### PR DESCRIPTION
**Pull request by ltln**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Twitter/vi.md [hunk 1]
```diff
@@ -1,11 +1,6 @@
----
-outdated_translation: true
-outdated_since: bf0efccf8db8b9dfb5ae645549a64c0623c27052
----
-
 # Twitter
 
-osu! có một vài tài khoản Twitter khác nhau, mỗi cái phục vụ cho mục đích cụ thể. Phần lớn trong số đó ít khi hoạt động nhưng chúng sẽ giúp bạn nắm bắt được thông tin ở ngoài hệ sinh thái osu!.
+osu! có một vài tài khoản Twitter khác nhau, mỗi tài khoản phục vụ cho mục đích cụ thể. Phần lớn trong số đó ít khi hoạt động nhưng chúng sẽ giúp bạn nắm bắt được thông tin ở ngoài hệ sinh thái osu!.
```
Comment (TranT3011) [inline wiki/Twitter/vi.md:3]:
```suggestion
osu! có nhiều tài khoản Twitter khác nhau, mỗi tài khoản phục vụ cho mục đích cụ thể. Phần lớn trong số đó ít khi hoạt động nhưng chúng sẽ giúp bạn nắm bắt được thông tin ở ngoài hệ sinh thái osu!.
```

File: wiki/Twitter/vi.md [hunk 2]
```diff
@@ -21,6 +16,7 @@ osu! có một vài tài khoản Twitter khác nhau, mỗi cái phục vụ cho
 | ![Ảnh đại diện của osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Kênh thông tin chính. |
 | ![Ảnh đại diện của BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Mua vui cho cộng đồng khi có vấn đề xảy ra. |
```
Comment (TranT3011) [inline wiki/Twitter/vi.md:17]:
```suggestion
| ![Ảnh đại diện của BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Tấu hài cho cộng đồng khi có vấn đề xảy ra. |
```

File: wiki/Twitter/vi.md [hunk 3]
```diff
@@ -21,6 +16,7 @@ osu! có một vài tài khoản Twitter khác nhau, mỗi cái phục vụ cho
...
 | ![Ảnh đại diện của NAT](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat) | Tin tức, thông báo và khảo sát cộng đồng bởi [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (không được điều hành bởi [đội ngũ osu!](/wiki/People/The_Team)). |
+| ![Ảnh đại diện của pp committee](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Thông báo về tính toán độ khó cho tất cả chế độ game, điều hành bởi [Uỷ ban Điểm Thành Tích](/wiki/People/Performance_Points_Committee). |
```
Comment (TranT3011) [inline wiki/Twitter/vi.md:19]:
```suggestion
| ![Ảnh đại diện của pp committee](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Thông báo về tính toán độ khó cho tất cả chế độ game, điều hành bởi [Nhóm chuyên về pp](/wiki/People/Performance_Points_Committee). |
```

Comment (TranT3011) [reply wiki/Twitter/vi.md:19]: yours and mine both sound weird tbh but i dont see a reason to use a word usually being used in government lol

**Review by TranT3011 — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Comment by TicClick** [comment]
for future PRs, please make a separate branch beforehand and don't work in `master`: https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits

**Pull request merged**